src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/ExpressRouteCircuitTests.ps1

```diff
@@ -140,46 +140,135 @@ function Test-ExpressRouteCircuitPeeringCRUD
+		Assert-AreEqual "30" $p.PeerASN
+		Assert-AreEqual "192.168.1.0/30" $p.PrimaryPeerAddressPrefix
+		Assert-AreEqual "192.168.2.0/30" $p.SecondaryPeerAddressPrefix
+		Assert-AreEqual "33" $p.VlanId
+		
+		#set public peering
+	    $circuit = Get-AzureRmExpressRouteCircuit -Name $circuitName -ResourceGroupName $rgname | Set-AzureRmExpressRouteCircuitPeeringConfig -Name AzurePublicPeering -PeeringType AzurePublicPeering -PeerASN 100 -PrimaryPeerAddressPrefix "192.168.1.0/30" -SecondaryPeerAddressPrefix "192.168.2.0/30" -VlanId 55  | Set-AzureRmExpressRouteCircuit 
+		$p = $circuit | Get-AzureRmExpressRouteCircuitPeeringConfig -Name AzurePublicPeering
+
+		Assert-AreEqual "AzurePublicPeering" $p.Name
+		Assert-AreEqual "AzurePublicPeering" $p.PeeringType
+		Assert-AreEqual "100" $p.PeerASN
+		Assert-AreEqual "192.168.1.0/30" $p.PrimaryPeerAddressPrefix
+		Assert-AreEqual "192.168.2.0/30" $p.SecondaryPeerAddressPrefix
+		Assert-AreEqual "55" $p.VlanId
+
+		$listPeering = $circuit | Get-AzureRmExpressRouteCircuitPeeringConfig
+		Assert-AreEqual 2 @($listPeering).Count			
+
+		# Delete Circuit
+        $delete = Remove-AzureRmExpressRouteCircuit -ResourceGroupName $rgname -name $circuitName -PassThru -Force
+        Assert-AreEqual true $delete
+		        
+        $list = Get-AzureRmExpressRouteCircuit -ResourceGroupName $rgname
+        Assert-AreEqual 0 @($list).Count
+		
+		
+    }
+    finally
+    {
+        # Cleanup
+        Clean-ResourceGroup $rgname
+    }
+}
+
+<#
+.SYNOPSIS
+Tests express route microsoft peering
+#>
+function Test-ExpressRouteCircuitMicrosoftPeeringCRUD
+{
+    # Setup
+    $rgname = Get-ResourceGroupName
+    $circuitName = Get-ResourceName
+	$rglocation = Get-ProviderLocation ResourceManagement
+    $resourceTypeParent = "Microsoft.Network/expressRouteCircuits"
+    $location = Get-ProviderLocation $resourceTypeParent
+    $location = "brazilSouth"
+    try 
+    {
+        # Create the resource group
+        $resourceGroup = New-AzureRmResourceGroup -Name $rgname -Location $rglocation
+        
+        # Create the ExpressRouteCircuit with peering
+		$peering = New-AzureRmExpressRouteCircuitPeeringConfig -Name MicrosoftPeering -PeeringType MicrosoftPeering -PeerASN 33 -PrimaryPeerAddressPrefix "192.168.1.0/30" -SecondaryPeerAddressPrefix "192.168.2.0/30" -VlanId 223 -MicrosoftConfigAdvertisedPublicPrefixes @("11.2.3.4/30", "12.2.3.4/30") -MicrosoftConfigCustomerAsn 1000 -MicrosoftConfigRoutingRegistryName AFRINIC 
+		$circuit = New-AzureRmExpressRouteCircuit -Name $circuitName -Location $location -ResourceGroupName $rgname -SkuTier Premium -SkuFamily MeteredData  -ServiceProviderName "equinix" -PeeringLocation "Silicon Valley" -BandwidthInMbps 1000 -Peering $peering
+        
+        #verification
+        Assert-AreEqual $rgName $circuit.ResourceGroupName
+        Assert-AreEqual $circuitName $circuit.Name
+        Assert-NotNull $circuit.Location
+        Assert-NotNull $circuit.Etag
```

Comment:
@dihan0604 please change the indentation in this file (replace Tabs to 4 spaces)